### PR TITLE
Fix panic triggering via HTTP API

### PR DESCRIPTION
**Pull request by paulcarlton-ww**
The GET /panic api call is not working due the the logger.Panic method failing to call panic. This change replaces the logger.Panic method call with logger.Info and adds a call to os.Exit(255).

### CODE REVIEW

File: pkg/version/version.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
 package version
 
-var VERSION = "6.1.4"
+var VERSION = "6.1.5"
```
Comment (stefanprodan) [inline pkg/version/version.go:3]: Please undo this change, the version bump is done at release time.

Comment (paulcarlton-ww) [reply pkg/version/version.go]: done

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @paulcarlton-ww
